app/assets/scripts/actions/index.js

```diff
@@ -213,6 +236,9 @@ export function getEruOwners () {
 
 export const GET_DISTRICTS = 'GET_DISTRICTS';
```

Comment:
Leaving a note for myself here to check conventions for action names and see if this could be better.